doc/analyzers/VSTHRD102.md

```diff
@@ -22,7 +22,7 @@ internal void DoWork()
 {
     joinableTaskFactory.Run(async delegate // Analyzer will flag this line
     {
-        await DoSomethingAsync();
+        await PublicMethodAsync();
```

Comment:
This would introduce infinite recursion, since `PublicMethodAsync` calls `DoWork`. 
The idea here was that `DoWork()` did something (anything!) asynchronously.